mmpose/apis/inference.py

```diff
@@ -145,6 +146,20 @@ def _inference_single_pose_model(model,
     if device.type == 'cpu':
         device = -1
 
+    # to support multi-frame setting, the imgs_or_paths is a list/tuple
+    # even if it has only one image/path
+    assert isinstance(imgs_or_paths, list)
+    if 'frame_weight_test' in cfg.data_cfg:
```

Comment:
It would be better if we can use the model name or something else to make a decision on running multi-frame or single-frame inference, because ``frame_weight_test`` is not a strong indicator.

Reply:
I have added a flag argument in `_inference_single_pose_model`. Could you please help me check this again?